pkg/actuators/machine/actuator.go

```diff
@@ -471,6 +476,11 @@ func (a *Actuator) Update(context context.Context, machine *machinev1.Machine) e
 		return fmt.Errorf("%s: failed to patch machine: %v", machine.Name, err)
 	}
 
+	err = a.requeueIfInstancePending(machine)
```

Comment:
this can be simplified by just  `a.requeueIfInstancePending(machine)`, no need for `if` statement